AzureHosted.md

```diff
@@ -1,10 +1,26 @@
-# Logic App-based Industry Link in Azure Marketplace
+# Azure hosted Industry Link
 
-This template demonstrates how to build an Industry Link using Logic Apps deployed to Azure. This is ideal for transferring data between any of the following data sources and sinks: API using a custom connector, files in Azure Blob Storage, Azure Event Hubs and Dataverse.
+## Overview
 
-The [MsIndustryLinks](scripts/modules/MsIndustryLinks/README.md) module supports the creation and configuration of Logic App workflow templates, ARM templates for deploying the Industry Link resources to Azure, and the application offer for publishing to the Azure Marketplace.
+This Industry Link solution option utilizes Azure to integrate your data with Microsoft Dataverse. A set of modular Logic Apps are created to facilitate the data transfer and basic transformation requirements between your data source and sink.
 
-## Install MsIndustryLinks module
+![Azure solution architecture](./images/AzureHostedArchitecture.png)
+
+The Logic Apps are packaged into an Azure deployment package which contains the ARM templates required to deploy the required Azure resources to your Azure environment. You can also make your Industry Link available for external customers by using this package as part of your offer when publishing to the Azure Marketplace.
+
+Once published to Partner Center, a customer is able to install the Industry Link template into their own Azure subscription with the following steps:
+1. Install the Industry Link into your Azure subscription via the offer in Azure Marketplace.
+2. Turn on the Logic App upon successful deployment.
```

Comment:
The Logic App should already be running in Azure after deployment. I'll need to double check and jog my memory.

Reply:
Yeah, let's remove step #2. Or maybe just add something along the lines of, customer can further customize any settings if needed (optional).

Reply:
While a customer can customise the Industry Link once its installed, it may not always be the recommendation from an ISV. Lets keep it simple with a single step for now.